crates/gosub_pipeline/viewpage.sh

```
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
+set -euo pipefail
+
+if [ -z "${1:-}" ]; then
+    echo "Usage: $0 <url>" >&2
+    exit 1
+fi
+
+. tools/souper/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: tools/souper/bin/activate was not specified as input (see shellcheck -x).

(SC1091)